devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -1435,13 +1457,16 @@ void YarpRobotLoggerDevice::recordVideo(const std::string& cameraName, VideoWrit
 
         // release the CPU
         BipedalLocomotion::clock().yield();
-
-        if (wakeUpTime < BipedalLocomotion::clock().now())
+        auto endTime = BipedalLocomotion::clock().now();
+        if (wakeUpTime < endTime)
         {
             log()->info("{} The video thread spent more time than expected to save the camera "
-                        "named: {}.",
+                        "named: {}. Expected: {}s. Measured: {}s. Nominal: {}s.",
```

Comment:
If I'm not mistaken, it is not required to add s for indicating second. Spdlog should do it automatically

Reply:
Thanks for noticing. Removed in https://github.com/ami-iit/bipedal-locomotion-framework/pull/910/commits/ccce521016da338b3ec832edd3cebe002a44bed6